packages/mandb.rb

```diff
@@ -54,10 +54,6 @@ def self.install
   end
 
   def self.postinstall
-    puts
-    puts "To finish the installation, set the default MANPATH environment variable:".lightblue
-    puts "echo \"export MANPATH=#{CREW_PREFIX}/man:$MANPATH\" >> ~/.bashrc".lightblue
```

Comment:
Yay! This is actually what I was trying to remove with #3002.

Reply:
Thanks to @superloach for following through on this.